.github/workflows/benchmarks.yml

```diff
@@ -127,23 +127,23 @@ jobs:
         if: always()
         run: make destroy
 
-      - if: failure()
+        # Notify failure to Slack only on schedule (nightly run)
+      - if: failure() && github.event_name == 'schedule'
```

Comment:
I think we also need a similar `- if: github.event_name == 'schedule'` on the step below, for the success case.

Reply:
It's done @axw